Formula/pipenv.rb

```diff
@@ -6,19 +6,7 @@ class Pipenv < Formula
   url "https://files.pythonhosted.org/packages/b5/3f/80a09ea5f279e90122600d24cfa25d698008dc7e5b261cb582de8966bbe2/pipenv-8.3.2.tar.gz"
   sha256 "20be245ad2a7908a04b302e9568ee26e57f8dacc558468d2a84e699b2622e9af"
 
-  bottle do
```

Comment:
The bottle block shouldn't be modified or deleted manually.

Reply:
I will re-add it.

Reply:
@ilovezfs Would the removal of the bottle block cause the CI failure?

Reply:
nope